### PR TITLE
community/phpldapadmin: modernize and bump dependencies to use php7

### PR DESCRIPTION
**Pull request by andypost**
Ref https://github.com/alpinelinux/aports/pull/5569

Patch from https://github.com/breisig/phpLDAPadmin/commit/db8a98c7a5ae6352018e2a2673e3d24df03f3e48

### CODE REVIEW

**Comment by andypost** [comment]
All dependencies updated - mbstring & openssl optional but their methods used all over code conditionally.
I used to test it with edge php7 
- copy `/etc/phpldapadmin/config.php` and add server's host
- the run build-in server `cd /usr/share/webapps/phpldapadmin/htdocs && php -S 127.0.0.1:8080`

**Comment by andypost** [comment]
@rnalrd it needs you approval to move ahead

File: community/phpldapadmin/APKBUILD [hunk 1]
```diff
@@ -1,17 +1,12 @@
 # Maintainer: Leonardo Arena <rnalrd@alpinelinux.org>
-_php=php5
```
Comment (rnalrd) [inline community/phpldapadmin/APKBUILD]: I'd prefer to do not remove it. Other packages use the same approach. It's handy for the next php version.

Comment (andypost) [reply community/phpldapadmin/APKBUILD]: I will revert template but package is broken cos at least php5-session is needed, olso I'd added opcache or apc in phpt

Comment (andypost) [reply community/phpldapadmin/APKBUILD]: Also I'm not sure zlib is bundled in php5 build like in php7

Comment (andypost) [reply community/phpldapadmin/APKBUILD]: Pushed updates

**Pull request closed without merging**